File: net/merge/Merger_pdf_pdfa.cs
using Aspose.Pdf;

namespace aspose_snippets.net
{
    public static partial class Merger
    {
        public static void pdf_to_pdfa()
        {
            var pathSource1 = @"..\..\TestData\test.pdf";
            var pathSource2 = @"..\..\TestData\Second\test.pdf";

            //read pdf file to Aspose Document
            var firstDoc = new Document(pathSource1);
            var secondDoc = new Document(pathSource2);

            //create empty pdf document
            var outputDoc = new Document();

            //set less memory usage with unload instead of fast performance
            outputDoc.EnableObjectUnload = true;

            foreach (var page in firstDoc.Pages)
            {
                //add page from one document to another directly
                outputDoc.Pages.Add(page);
            }

            foreach (var page in secondDoc.Pages)
            {
                //add page from one document to another directly
                outputDoc.Pages.Add(page);
            }

            //save document as specific pdf standard PDFA 3Y
            outputDoc.Convert(
                "test.pdf", 
                PdfFormat.PDF_A_3U, 
                //delete objects that impossible to convert
                ConvertErrorAction.Delete);
        }
    }
}

File: java/merge/src/aspose_snippets/java/Merger_pdf_pdfa.java
package aspose_snippets.java;

public class Merger_pdf_pdfa {
    public static void Execute() {
        String pathSource1 = "../../TestData/test.pdf";
        String pathSource2 = "../../TestData/Second/test.pdf";

        //read pdf file to Aspose Document
        com.aspose.pdf.Document firstDoc = new com.aspose.pdf.Document(pathSource1);
        com.aspose.pdf.Document secondDoc = new com.aspose.pdf.Document(pathSource2);

        //create empty pdf document
        com.aspose.pdf.Document outputDoc = new com.aspose.pdf.Document();

        //set less memory usage with unload instead of fast performance
        outputDoc.setEnableObjectUnload(true);

        for (com.aspose.pdf.Page page : firstDoc.getPages()) {
            //add page from one document to another directly
            outputDoc.getPages().add(page);
        }

        for (com.aspose.pdf.Page page : secondDoc.getPages()) {
            //add page from one document to another directly
            outputDoc.getPages().add(page);
        }

        //save document as specific pdf standard PDFA 3Y
        outputDoc.convert(
                "Merger_pdf_pdfa",
                com.aspose.pdf.PdfFormat.PDF_A_3U,
                //delete objects that impossible to convert
                com.aspose.pdf.ConvertErrorAction.Delete);
    }
}
